src/main/java/no/fdk/searchapi/controller/datasetssearch/DatasetsSearchQueryBuilder.java

```diff
@@ -255,5 +255,10 @@ static QueryBuilder keywords(String value, DatasetsSearchQueryBuilder queryBuild
             });
             return boolQueryBuilder;
         }
+
+        static QueryBuilder uris(String value, DatasetsSearchQueryBuilder queryBuilder) {
+            String[] uris = value.split(",");
+            return QueryBuilders.termsQuery("uri", uris);
+        }
```

Comment:
Consider this far more elegant and superior-in-every-way solution
```suggestion
        static QueryBuilder uris(String value, DatasetsSearchQueryBuilder queryBuilder) {
            return QueryBuilders.termsQuery("uri", value.split(","));
        }
```